### PR TITLE
Replace internal config._origargs with invocation_params.args

### PR DESCRIPTION
**Pull request by blueyed**
Added in https://github.com/pytest-dev/pytest/pull/5564.

### CODE REVIEW

File: src/_pytest/config/__init__.py [hunk 1]
```diff
@@ -915,7 +915,7 @@ def parse(self, args, addopts=True):
         assert not hasattr(
             self, "args"
         ), "can only parse cmdline args at most once per Config object"
-        self._origargs = args
+        assert self.invocation_params.args == args
```
Comment (blueyed) [inline src/_pytest/config/__init__.py]: Just for testing / ensuring - can be removed, and/or improved (msg).

Comment (blueyed) [reply src/_pytest/config/__init__.py]: @RonnyPfannschmidt / @nicoddemus 
What about this?

Comment (RonnyPfannschmidt) [reply src/_pytest/config/__init__.py]: hmm, that one seems ok to me

Comment (nicoddemus) [reply src/_pytest/config/__init__.py]: Agreed, nothing fishy at first glance.

Comment (blueyed) [reply src/_pytest/config/__init__.py]: I think it should have either be removed (I've added it for a full test run on CI), or improved (i.e. a msg).
But ok then for now.

**Pull request merged**